packages/expandable-card/src/ExpandableCard/ExpandableCard.tsx

```diff
@@ -55,6 +52,14 @@ const ExpandableCard = ({
 
   const transitionRef = useRef<HTMLElement | null>(null);
 
+  const previousDefaultOpen = usePrevious(defaultOpen);
+
+  useEffect(() => {
+    if (previousDefaultOpen !== defaultOpen) {
+      setIsOpen(defaultOpen);
+    }
+  }, [defaultOpen, previousDefaultOpen]);
+
```

Comment:
@bruugey Can we move this to a separate PR?

Reply:
yeah i think i must have just caught it here